.github/workflows/unit.yml

```diff
@@ -46,7 +58,7 @@ jobs:
             - name: Install the dependencies
               env:
                   SYMFONY_REQUIRE: ${{ matrix.symfony }}
-              run: composer install --no-interaction --prefer-dist --no-progress --no-suggest
+              run: composer install --no-interaction --prefer-dist --no-progress --no-suggest --no-plugins
```

Comment:
using `--no-plugins` means that `SYMFONY_REQUIRE` won't be respected as Flex won't be loaded. This is not the right fix to allow the globally installed plugin. We install it because we want to use it.

Reply:
Ok, I wasn't sure how to fix it, I change to add flex plugin